src/mame/barcrest/mpu4.cpp

```diff
@@ -2536,8 +2626,8 @@ void mpu4_state::mod4yam(machine_config &config)
 	MCFG_MACHINE_START_OVERRIDE(mpu4_state,mpu4yam)
 
 	mpu4_reels<0, 6>(config);
-
-	YM2413(config, m_ym2413, 2000000);
+	 
+	YM2413(config, m_ym2413, XTAL(3'579'545)/2); // XTAL on sound board
```

Comment:
I doubt this divider is real. The datasheet says a 3.579545 MHz should be connected directly to pins 8 and 9, and it usually is on other hardware.

Reply:
alright, I've removed the divider